app/javascript/packages/address-search/package.json

```diff
@@ -1,25 +1,29 @@
 {
   "name": "@18f/identity-address-search",
-  "private": true,
-  "version": "1.0.0",
+  "version": "1.0.3",
```

Comment:
Why are there already published versions? 🤔 Do we need to bump this version again if we're going to publish another one?